scripts/pi-hole/php/header.php

```diff
@@ -220,6 +220,10 @@
                  <li>
                     <a href="https://docs.pi-hole.net/" rel="noopener" target="_blank"><i class="fa fa-fw fa-question-circle"></i>&nbsp;&nbsp;Documentation</a>
                 </li>
+                <!-- API Docs -->
+                <li>
+                   <a href="/api/docs/" rel="noopener" target="_blank"><i class="fa fa-fw fa-question-circle"></i>&nbsp;&nbsp;API Documentation</a>
```

Comment:
Is there any better way to handle the horizontal placement besides sticking in to nbsp's?

Reply:
Untested, try this:

```suggestion
                   <a href="/api/docs/" rel="noopener" target="_blank"><i class="fa fa-fw fa-question-circle"></i><span>API Documentation</span></a>
```

Reply:
`<span>` did not work
![Bildschirmfoto zu 2021-04-25 20-38-07](https://user-images.githubusercontent.com/26622301/116005214-3db1a900-a606-11eb-9133-34bdcd03bb58.png)

Reply:
`<span style="padding-left: 2px;">API Documentation</span></a>` does work.

Taken from https://www.freecodecamp.org/news/tab-space-instead-of-multiple-non-breaking-spaces/

Should I replace it for all items in the side bar?

Reply:
Probably would be better to assign a CSS class to it and then only have to update the CSS and not remember to style a handful of spans when and if they change.

Reply:
As a point to note: The plan is to use something like Vue or an actual structured interface instead of AdminLTE for v6. Or else update AdminLTE to Bootstrap 4 and not keep patching an ancient version.

Reply:
Is it worth then to add a css class and change in in all items instead of add a single element the same way the rest is done now?

Reply:
I guess it's a matter of wanting to write clean and good code or just wanting something that works now and is lazy.

Reply:
I would offer that it's more a matter of what we have the resources for. @yubiuser, double `&nbsp` fine for now if the `<span>` element isn't working. At this stage it's a link going into an alpha version of a web interface that will serve as a reminder to include that link in the final version, no matter what it's final form is, which can come in a later PR.

Reply:
I don't agree.

This wasn't a PR that had to go in quickly. We could have taken the time to add this properly and cleaned up the code in the process. Now we've just added in more things to fix later and added more to the list of work that will need to be done some time. Just because the other links are done poorly isn't a reason to continue to do the same.

I wouldn't have requested this if I didn't think Yubi was capable of adding the link and improving the quality of the code. 

And please, in the future, don't approve and merge code when there's still an open discussion that needs resolution. I don't shut you all down like that, don't shut me down like that.

Reply:
https://github.com/pi-hole/AdminLTE/pull/1789

Reply:
Thanks!